### ISSUE TITLE
Rate limiter is consuming all errors thrown by axios

### ISSUE THREAD

**Issue by mcuppi** [aishek/axios-rate-limit#1]
The rate limiter intercepts requests and responses from axios and doesn't re-throw errors. This causes any errors thrown by axios to be consumed (i.e., you cannot catch them later on in a try catch block).
  
This can be fixed by simply adding error handling to the interceptors:
```
AxiosRateLimit.prototype.enable = function (axios) {
  this.interceptors.request = axios.interceptors.request.use(
    this.requestHandler,
    function (error) {
      return Promise.reject(error)
    }
  )
  this.interceptors.response = axios.interceptors.response.use(
    this.responseHandler,
    function (error) {
      return Promise.reject(error)
    }
  )
}
```

I've submitted a pull request that addresses this issue: #2

**Comment by aishek** [comment]
Merged the pull request.

**Comment by aishek** [comment]
Published in version `0.0.4`. Thanks!
